kuma/attachments/jinja2/attachments/includes/attachment_list.html

```diff
@@ -105,6 +105,8 @@
               <a href="{{ document_attachment.document.get_absolute_url() }}">
                 {{ document_attachment.document.title }} ({{ document_attachment.document.locale}})
               </a>{% if not loop.last %}, {% endif %}
+            {% else %}
+              {{ _('n/a') }}
```

Comment:
I wasn't able to test this branch, because I can't find any examples in my database. As far as I can tell, the case to trigger it would be something like:

1) Create the sole translation of an English document, with an attachment
2) Delete the English version of the document
3) Edit the translated version

So, saying "looks good" without practical testing.